src/header.rs

```diff
@@ -978,6 +980,9 @@ pub enum DigestItemRef<'a> {
         opaque: &'a [u8],
     },
 
+    /// Some other thing. Unsupported and experimental.
```

Comment:
```suggestion
    /// Some other thing. Always ignored.
```

I don't think it's true that it's "unsupported and experimental", but I'd like to explicitly mention that it's ignored, in order to make it clear that it's not a good idea to put something consensus-critical in there, similar to what Substrate says.